docs/cdn_commands.md

```diff
@@ -102,3 +102,10 @@ cwr('recordEvent', {type: 'your_event_type', data: {field1: 1, field2: 2}})
 ```
 awsRum.recordEvent('your_event_type', {field1: 1, field2: 2})
 ```
+
+## RecordEventOptions
+
+| Field Name | Type | Default | Example | Description |
+| --- | --- | --- | --- | --- |
+| isCandidate | Boolean | FALSE | In this example, RUM will cache all 'running_max' candidates, but only call the final value=15 to PutRumEvents at the end of the Page's lifecycle. This is a very frugal solution if you need to keep a running count, but only want to publish the event at the latest possible time. <br/><br/>aws.recordEvent('final_click_count', {value: 5}, {isCandidate: true, replaceFirstMatch: true})<br/>aws.recordEvent('final_click_count', {value: 10}, {isCandidate: true, replaceFirstMatch: true})<br/>aws.recordEvent('final_click_count', {value: 15}, {isCandidate: true, replaceFirstMatch: true}) | If true, then RUM will not dispatch the candidate event to PutRumEvents until the next visibilitychange event, which only occurs when the page is hidden or closed.<br/><br/>Event candidates are prioritized over regular events in that final batched call because that is the only opportunity for them to be reported. <br/><br/>To provision more cache space for candidates, then increase candidatesCacheSize. |
```

Comment:
Could you also format the examples? e.g:

`aws.recordEvent('final_click_count', {value: 5}, {isCandidate: true, replaceFirstMatch: true})`

Reply:
In revision 2, I removed made `recordCandidate()` only available for internal plugins so the public documentation is no longer necessary. If we want, we can always make it public later.